platform/accessibility/README.md

```diff
@@ -1,98 +1,63 @@
- * [508 checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md)
- * [508, accessibility, and inclusive design best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md).
-* **Define:** We help in early project discussions to make sure inclusive design is woven into your work. Main areas to consider at this phase include:
-   * challenges with user flows
-   * color contrast
-   * color blindness
-   * accessible interactions
-   * design for various device sizes
-   * cognitive impact
-   * considerations for including diverse research participants
-* **Discover:** We provide more specific feedback at this phase, including:
-   * relevant design system components and patterns
-   * feedback on recruiting diverse research participants
-   * additional testing scenarios
-* **Build:** We conduct the following accessibility checks:
-   * Review your completed test plan and automated end-to-end test results
-   * Manually test the [happy path](https://en.wikipedia.org/wiki/Happy_path)
-   * Triage issues for severity and whether they must be fixed pre- or post-launch. This testing is done as part of the VSP staging review.
-* **Validate:** Post launch, we do a full accessibility audit and ask the VA 508 office to conduct a review:
-   * Post-launch audits uncover usability issues that might not otherwise appear on an automated report.
-   * These findings drive future innovation, updates to the design system, and improve research sessions. (Note: This audit was previously done prior to launch, but is now done post-launch to emphasize speed and agility.)
-
-## Resources and guides
-**General Guidance**
+
+Documentation about accessibility best practices is always available to VFS teams, as are various automated and manual tools for accessibility testing.
+
+VFS teams also receive guidance and assessments from the DEPO Platform as a part of the [Collaboration Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/working-with-vsp/vsp-collaboration-cycle), which is the process for interacting with DEPO Platform personnel to ensure your [VA.gov](https://www.va.gov/) product meets or exceeds DEPO's standards.
+
+As a part of the collaboration cycle, VFS teams must do foundational accessibility testing and resolve or document any resulting issues prior to the staging review.  Teams with their own accessibility experts may be expected to do advanced accessibility testing prior to the staging review. The DEPO Platform will spot check products to catch major issues prior to launch, and will assign issues to the VFS team for resolution. Post-launch is when the VA 508 Office will do a more robust review of the product, and you'll be responsible for resolving any additional issues that are found.
```

Comment:
What do you think about linking the text "foundational accessibility testing" to the staging review document ( https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md ) initially, and then updating this link post-PR to point at the foundational testing heading.

Reply:
Great idea, updated.